src/base/cubrid_getopt_long.c

```diff
@@ -36,7 +36,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
-#include "getopt.h"
+#include "cubrid_getopt.h"
```

Comment:
1. if you renamed getopt.h -> cubrid_getopt.h then I suggest renaming the implementation accordingly:
getopt_long.c -> cubrid_getopt.c, looks natural

2. the code from cubrid_getopt.h is protected by HAVE_GETOPT_H and this file  is guarded by HAVE_GETOPT_LONG ... and contains implementation for what's declared in header
Better to be consistent and protect both files with the same HAVE_GETOPT_H

Other solution would be to split getopt() and getopt_long() in 2 separate sets of .h/.c files but doesn't make too much sense because these functions go hand in hand anyway

Reply:
regarding first point, I tried to keep close as much as possible to current structure, therefore I did these 2 renamings:
getopt.h => cubrid_getopt.h
getopt_long.c => cubrid_getopt_long.c